layout/theme.liquid

```diff
@@ -123,7 +123,7 @@
           </div>
         </div>
         {% else %}
-          {% unless current_user.is_anonymous? %}
+          {% if current_user.is_wholesale? %}

```

Comment:
@cecilia-ombulabs No, this is wrong. We want to show the admin bar for all users except the anonymous users.